requirements/dev.txt

```diff
@@ -66,16 +66,16 @@ beautifulsoup4==4.8.2 \
     --hash=sha256:05fd825eb01c290877657a56df4c6e4c311b3965bda790c613a3d6fb01a5462a \
     --hash=sha256:9fbb4d6e48ecd30bcacc5b63b94088192dcda178513b2ae3c394229f8911b887 \
     --hash=sha256:e1505eeed31b0f4ce2dbb3bc8eb256c04cc2b3b72af7d551a4ab6efd5cbe5dae
-boto3==1.11.10 \
-    --hash=sha256:27e221d3868f35687807e5c920f7e8d4872f722f64196a7fd274a06ad65beec0 \
-    --hash=sha256:8ff4e3d9e5d6a26dd7494afc68dc96afe6b7bda88130cca84cd58702d888ed27 \
-    # via aws-sam-translator, moto
+boto3==1.9.240 \
+    --hash=sha256:0895805810df8cc49898fbeb61479c20c63a59fb6d3adf19f78fde90efec10a2 \
+    --hash=sha256:2f174f1e01d898ed940677a66404b7d30cbe92665b1fa2a5086eb64382ce2c41
```

Comment:
You should `git checkout upstream requirements/*.txt; tools/update-locked-requirements` to get rid of these downgrades.

Reply:
Cool thanks I was just looking at that and wondering.